LACP - Resilience update changes

SlimJ4 opened the pull request:
<!--
Hello 👋
Thank you for submitting a Pull Request.

For Work In Progress Pull Requests, please use the Draft PR feature, see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details. As long as your Pull Request is a draft, the OVHcloud Guides Team will not start proofreading it.

For a timely review/response, please avoid force-pushing additional commits if your PR already received reviews or comments.

DO NOT USE a fork if you are an OVHcloud employee. Instead, please contact the OVHcloud Guides Team so to be granted /write access to this repository.

Feel free to apply labels on your Pull Request, selecting them from the list to your right.

Before submitting a Pull Request, please ensure you have:

- 📖 Read the OVHcloud Documentation readme: https://github.com/ovh/docs/blob/develop/readme.md
- 🇬🇧 Edited (at least) the en-gb version of a guide.
- 🇫🇷 Edited also the fr-fr version if you are a French speaker.
- 🌐 Edited all the files if you're doing a fix or a minor update (this is not mandatory but recommended).
- 👷‍♀️ Created a small PR, if applicable.
- ✅ Tested every step you're documenting.
- 📝 Used descriptive commit messages.
- 📐 Resized images which width exceeds 1400px.
- 📝 Edited or blurred any sensitive/private information from your text and images (IPs, user IDs, private URLs, etc.).
-->

## What type of Pull Request is this?

- Update of existing guide(s)

## Description

Updates several guides to explain some changes and add precisions related to the resilience update in the Link Aggregation configuration algorithm.

## Mandatory information

The translations in this Pull Request have been done using:
- [ ] OVHcloud integrated translation LLM <!-- If you're interested in this new option, contact the Guides team -->
- [ ] Systran
- [X] Other tool: none
- [ ] This Pull Request didn't require any translation.

<!-- Delete the line which doesn't apply: -->
- This Pull Request can be merged as soon as possible. <= Currently under review

SlimJ4 committed:
bond public interface

SlimJ4 committed:
typo fix

SlimJ4 committed:
netplan + ifupdown + rescue

SlimJ4 committed:
netplan + ifupdown + rescue

sbraz commented on pages/bare_metal_cloud/dedicated_servers/lacp-enable-netplan/guide.en-gb.md:
```diff
@@ -43,7 +49,10 @@ Switch to the tab `Network Interfaces`{.action} and take note of the MAC address
 
 > [!primary]
-> Please note that the MAC address of the **main** public interface is the one with the lowest value. In the example image above, this is the address `a1:b2:c3:d4:e5:c6`.
+> Please note that the **main public** interface is the one answering to DHCP requests.
```
Il manque des détails que tu as sur la traduction française.

sbraz commented on pages/bare_metal_cloud/dedicated_servers/rescue_mode/guide.en-gb.md:
```diff
@@ -352,6 +352,14 @@ chroot /mnt/
+
+Link Aggregation (LACP) is highly beneficial, as it increases your server's total bandwidth while providing network redundancy in case a network interface fails.
+
+Although rescue mode is based on the Debian 12 operating system, its network configuration relies on the *ifupdown* utility.
```
Est-ce qu'il faut dire "rescue mode" ou "the rescue mode" ?

SlimJ4 replied:
Dans ce cas précis, "rescue mode", vu qu'on parle du concept (zero article rule): https://www.grammarwiz.com/zero-article-rules-with-examples.html
Merci de m'avoir fait me poser la question, je l'ai écrit instinctivement sans connaître la règle.

sbraz commented on pages/bare_metal_cloud/dedicated_servers/ola-enable-debian9/guide.fr-fr.md:
```diff
@@ -24,9 +24,16 @@ La technologie OVHcloud Link Aggregation (OLA) est conçue par nos équipes pour
 ## En pratique
 
-Étant donné que nous avons une configuration privée-privée pour nos NIC sur OLA, il est impossible de se connecter en SSH au serveur. Par conséquent, vous devrez utiliser l’outil IPMI pour accéder au serveur.
+Étant donné que nous avons une configuration privée-privée pour nos NICs sur OLA, il est impossible de se connecter en SSH au serveur. Par conséquent, vous devrez utiliser l’outil IPMI pour accéder au serveur.
```
Il ne faut pas de s à NIC, les sigles sont invariables : https://www.antidote.info/fr/blogue/enquetes/pour-les-cingles-des-sigles-et-les-accros-des-acronymes#pluriel

SlimJ4 replied:
Noté ! J'ai fait un raccourci, à cause de l'anglais qui va privilégier "NICs".

sbraz commented on pages/bare_metal_cloud/dedicated_servers/lacp-enable-netplan/guide.fr-fr.md:
Tu peux ajouter une mention du fait que le rescue aussi reçoit les offres DHCP sur cette interface ? Il n'y a pas que l'OS installé sur disque.

sbraz reviewed (dismissed):
J'ai poussé quelques changements. Je te laisse regarder les commentaires, il reste quelques chose que j'aimerais changer.

sbraz commented on pages/bare_metal_cloud/dedicated_servers/ola-enable-debian9/guide.en-gb.md:
```diff
@@ -24,6 +24,13 @@ OVHcloud Link Aggregation (OLA) technology is designed by our teams to increase
 
+> [!primary]
+>
+> This guide provides instructions for configuring network interface bonding specifically using `ifupdown`. It also applies to the rescue system.
```
Préciser que ifupdown = /etc/network/interfaces car l'utilisateur ne connaît souvent que ce nom

SlimJ4 committed:
Merge branch 'sa-lacp-mac' of https://github.com/ovh/docs into sa-lacp-mac

SlimJ4 committed:
requested edits

Edited following recommendations from @sbraz .

Commit:
ola: reword some parts

SlimJ4 committed:
Disclaimer edit

Changed the disclaimer on the Netplan guide to include the case where the user installed Debian with a custom image.

SlimJ4 committed:
typo

sbraz commented on pages/bare_metal_cloud/dedicated_servers/lacp-enable-netplan/guide.fr-fr.md:
```diff
@@ -22,7 +22,16 @@ details[open]>summary::before {
+> Bien que les images Debian 12 (ou plus récentes) fournies par OVHcloud utilisent `Netplan` par défaut, il existe deux exceptions clés où `ifupdown` (/etc/network/interfaces) est utilisé à la place :
+>
+> - **Mode rescue** : Bien qu'il soit basé sur Debian 12, le mode rescue s'appuie sur l'utilitaire `ifupdown`.
+> - **Images personnalisées** : Les installations Debian effectuées à l'aide de votre propre image peuvent toujours utiliser `ifupdown` pour leur configuration réseau.
```
```suggestion
> - **Images personnalisées** : Il se peut que les installations Debian effectuées à l'aide de votre propre image utilisent toujours `ifupdown` pour leur configuration réseau.
```

sbraz committed:
Apply suggestion from @sbraz

Jessica41 committed:
update date and duplos

Jessica41 committed:
fix

benchbzh committed:
date update

Y0Coss commented:
Thank you for your contribution @SlimJ4
